### PR TITLE
Cleanup some of the badges on Readme

### PR DESCRIPTION
**Pull request by mrunalk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,31 +27,8 @@ These plug-ins simplify integration of the Rollbar.NET Notifier into codebases t
 [![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=rollbar-net)
 [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=rollbar-net)
 [![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=vulnerabilities)](https://sonarcloud.io/dashboard?id=rollbar-net)
-[![Bugs](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=bugs)](https://sonarcloud.io/dashboard?id=rollbar-net)
```
Comment (akornich) [inline README.md:30]: why are we removing the Bugs badge? these are not actual bugs filed against the repo but SonarCloud's potential buggy code indicator/counter...

File: README.md [hunk 2]
```diff
@@ -27,31 +27,8 @@ These plug-ins simplify integration of the Rollbar.NET Notifier into codebases t
...
-[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=ncloc)](https://sonarcloud.io/dashboard?id=rollbar-net)
-[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=rollbar-net)
```
Comment (akornich) [inline README.md:32]: Duplicated Lines of code badge is important code quality indicator. BTW, once I remove deprecated code this badge is expected to show 0% or fractionally close to it

File: README.md (hunk 1 above)
Comment (mrunalk) [reply README.md:30]: One can always see these stats on SonarCloud page. I am just trying to simplify the readme page and keep only important badges in there. Personally I would prefer 3 SonarCloud badges but we can keep 5 for now.

File: README.md (hunk 2 above)
Comment (mrunalk) [reply README.md:32]: As explained above one can always see additional stats on SonarCloud page. I think keeping Security, Reliability, Maintainability, Vulnerabilities and Coverage is good enough to for readme.

File: README.md (hunk 1 above)
Comment (akornich) [reply README.md:30]: Do not be afraid of badges. They are very visual and attract one's attention. For example, https://github.com/badges/shields

**Comment by mrunalk** [comment]
Thanks @coryvirok, removed the Codacy badge.

**Pull request merged**